scripts/events_module/relationship/romantic_events.py

```diff
@@ -203,7 +203,7 @@ def start_interaction(cat_from, cat_to):
         relationship.used_interaction_ids.append(chosen_interaction.id)
 
         # affect relationship - it should always be in a romantic way
-        value_change = "increase" if positive else "decrease"
+        value_change = True if positive else False
```

Comment:
```suggestion
        value_change = "increase" if positive else "decrease"
```
`value_change` is used later for some strings, so let's keep this and just pass `positive` into `interaction_affect_relationships` as-is, since it's already a boolean.